repository/p.json

```diff
@@ -447,6 +447,19 @@
 				}
 			]
 		},
+		{
+			"name": "Paste As Column",
+			"description": "Paste clipboard content in block mode (like emacs rectangle mode)",
+			"details": "https://github.com/TheClams/PasteAsColumn",
+			"issues": "https://github.com/TheClams/PasteAsColumn/issues",
```

Comment:
This is the default location for issues, do you don't have to provide it here. You can also leave out the description, it'll take that from your GitHub repo.